draft-ietf-rtcweb-jsep.xml

```diff
@@ -407,6 +407,53 @@ setRemote(ANSWER) |                                   |
         sort of SessionDescription to produce.</t>
       </section>
 
+      <section title="Session Description control" anchor="sec.session-description-ctrl">
+	<t>JSEP defines the following objects as an API to allow the
```

Comment:
I would make this a continuation of the previous section:

In order to give the application control over various common session parameters, JSEP provides control surfaces which tell the browser how to generate session descriptions. This avoids the need for manual modification of session descriptions in most cases.